### PR TITLE
new PAML version. Copy extra binaries. Extended sanity checking

### PR DESCRIPTION
**Pull request by pescobar**
upgraded to latest version 4.8

added two new binaries "infinitesites" and "mcmctree" to files_to_copy

extended sanity checking

@fgeorgatos why did you use the option toolchainopts = {'optarch': False}

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 15
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/568f842f873852c1facb for a full test report.

File: easybuild/easyconfigs/h/HPCBIOS_Bioinfo/HPCBIOS_Bioinfo-20130829-ictce-5.3.0.eb [hunk 1]
```diff
@@ -56,7 +56,7 @@ dependencies = [
     ('mpiBLAST', '1.6.0'),
     ('NEURON', '7.2'),
     ('Oases', '0.2.08'),
-    ('PAML', '4.7'),
+    ('PAML', '4.7a'),
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/h/HPCBIOS_Bioinfo/HPCBIOS_Bioinfo-20130829-ictce-5.3.0.eb]: I guess you have an interesting reason for your diff, yet I'm inclined to ask what happened to this!

**Comment by fgeorgatos** [comment]
put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**